framework/logging.cpp

```diff
@@ -1788,7 +1791,7 @@ void KeyValuePairLogger::print(int tc)
         logging_printf(LOG_LEVEL_VERBOSE(1), "%s_random_generator_state = %s\n", test->id,
```

Comment:
I'd completely disable all but the YAML logger for a non-CPU device. That way, you don't have to worry about porting them.